kde-apps/libksieve/libksieve-25.07.90.ebuild

```diff
@@ -21,7 +21,7 @@ IUSE="speech"
 RESTRICT="test"
 
 RDEPEND="
-	dev-libs/cyrus-sasl
+	dev-libs/cyrus-sasl:2
```

Comment:
While that is fine to do, it was also fine not to declare the slot in the absence of any other slot available (for as long as I remember).

Reply:
Indeed (though I don't object to adding it either).

Reply:
sure. what matters more is pkgcheck, which is silent here, though.